### PR TITLE
added first version of challenge

### PR DESCRIPTION
**Pull request by ripcurlz**
players have to connect to TCP port 8080 with either e.g. netcat, telnet or sth. similar. They have to respond with the correct answer to the origin of the provided md5 hash.

if they get the answer right, they are greeted by a nyancat cli, every 10th second the flag is displayed in there.

### CODE REVIEW

File: Misc/nyaned_enough/challenge.pl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+$| = 1;
+
+print "5badcaf789d3d1d09794d8f021f40f0e\n";
+print "your answer?\n";
+
+$line = <STDIN>; 
+
+chomp $line;
+
+# 5badcaf789d3d1d09794d8f021f40f0e is the md5 of the word "starwars"
```
Comment (B-Schwarz) [inline Misc/nyaned_enough/challenge.pl:16]: This repo is public so better change it ^^

**Review by B-Schwarz — CHANGES_REQUESTED** [review]
Please change the hash and don't comment the solution as this repo is public

Otherwise it worked

File: Misc/nyaned_enough/challenge.pl (hunk 1 above)
Comment (ripcurlz) [reply Misc/nyaned_enough/challenge.pl]: changed it

**Review by B-Schwarz — APPROVED** [review]
I now see my problem with the earlier change request 🤣 

As the password is in clear at the if condition the changing of the hash is irrelevant, so I could've approved the first pull request ^^

**Pull request merged**